zaas-package/src/main/resources/bin/start.sh

```diff
@@ -195,6 +195,38 @@ ADD_OPENS="--add-opens=java.base/java.lang=ALL-UNNAMED
         --add-opens=java.base/sun.nio.ch=ALL-UNNAMED
         --add-opens=java.base/java.io=ALL-UNNAMED"
 
+get_enabled_protocol_limit()
+{
+    target=$1
+    type=$2
+    key_component=ZWE_configs_zowe_network_${target}_tls_${type}Tls
+    key_zowe=ZWE_zowe_network_${target}_tls_${type}Tls
+    echo ${!key_component:-${!key_zowe:-}}
+}
+
+get_enabled_protocol()
+{
+    target=$1
+    enabled_protocols_min=$(get_enabled_protocol_limit "${target}" "min")
+    enabled_protocols_max=$(get_enabled_protocol_limit "${target}" "max")
+
+    if [ "${enabled_protocols_min:-}" == "${enabled_protocols_max:-}" ]; then
+        echo "${enabled_protocols_max:-}"
+    elif [ -z "${enabled_protocols_min:-}" ]; then
+        echo "${enabled_protocols_max:-}"
+    else
+        enabled_protocols_max=${enabled_protocols_max:-TLSv1.3}
+
+        enabled_protocols=,SSLv3,TLSv1,TLSv1.1,TLSv1.2,TLSv1.3,TLSv1.4,
+        enabled_protocols=${enabled_protocols%%,${enabled_protocols_max}*}
+        enabled_protocols=${enabled_protocols#*${enabled_protocols_min},}
+        if [ ! -z ${enabled_protocols} ]; then
+          enabled_protocols=",${enabled_protocols}"
+        fi
+        echo "${enabled_protocols_min}${enabled_protocols},${enabled_protocols_max}"
```

Comment:
Is this also printing it to console?

Reply:
it is, was added by @pj892031 , but probably it's not necessary, was for debugging purpose I guess?

Reply:
I guess this was done as a way to send the string as a return value of the function, but printing in the console is not ideal

Reply:
yes that's to return the string. I can rather use a global variable so it won't be printed